### PR TITLE
worked on property pages

### PR DESCRIPTION
**Pull request by Omams21**

### CODE REVIEW

File: Desktop/property-pro-app/src/pages/Buy/Buy.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import "./buy.css";
+import Footer from "../../components/Footer/Footer";
+import Navbar from "../../components/navbar/Navbar";
+
+const Buy = () => {
+  return (
+    <>
+      <Navbar />
+      <h1>Property For Sale</h1>
+      <div className="property_wrapper">
```
Comment (Pauliski) [inline Desktop/property-pro-app/src/pages/Buy/Buy.js:12]: I think you should change this from snake-case

File: Desktop/property-pro-app/src/pages/Rent/Rent.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+
+const Rent = () => {
+  return <div>
+      <Navbar/>
+
+<h1>Property For Rent</h1>
+      <div className="property_wrapper">
+      {propertyData.map((property) =>
```
Comment (Pauliski) [inline Desktop/property-pro-app/src/pages/Rent/Rent.js:19]: I think this map item is supposed to have a unique key, try to  work with the browser console for error

File: Desktop/property-pro-app/src/pages/Rent/Rentproperty.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          <div className=" House-description ">
+            <div className=" house-title">
+              <p className="text-icon"></p>
+              <p className=" house-type"> {houseType}</p>
+            </div>
+
+            <p className="amount">{price}</p>
+            <p className="house-specification ">
+              <span className="bold">{bathNo}</span>Bath
+              <span className="bold">{bedNo}</span>Bed
```
Comment (Pauliski) [inline Desktop/property-pro-app/src/pages/Rent/Rentproperty.js:28]: you might want to look at giving your HTML element a descriptive className

**Review by Pauliski — CHANGES_REQUESTED** [review]
Nice work! just need some little touches

**Review by Shukura2 — APPROVED** [review]
Cool

**Pull request merged**